tests/js/spec/components/dropdownAutoCompleteMenu.spec.jsx

```diff
@@ -83,18 +84,18 @@ describe('DropdownAutoCompleteMenu', function () {
       </DropdownAutoCompleteMenu>
     );
 
-    wrapper.find('AutoCompleteItem').last().simulate('click');
+    userEvent.click(screen.getByRole('option', {name: 'Australia'}));
 
     expect(mock).toHaveBeenCalledTimes(1);
     expect(mock).toHaveBeenCalledWith(
       {index: 1, ...countries[1]},
-      {highlightedIndex: 0, inputValue: '', isOpen: true, selectedItem: undefined},
+      {highlightedIndex: 1, inputValue: '', isOpen: true, selectedItem: undefined},
```

Comment:
`highlightedIndex` changed to 1 with RTL which is the correct behavior (and a good example of `user-event` being useful)